utils.py

```diff
@@ -0,0 +1,164 @@
+"""A collection of utility functions (mostly file reading/writing) used
+in the PathCORE analysis scripts"""
+
+import os
+
+import pandas as pd
+
+
+def load_expression_dataset(path_to_file):
+    """
+    Parameters
+    -----------
+    path_to_file : str
+        The path to the expression dataset.
+        Currently expects a tab-delimited file with row and column names,
+        where the rows are the genes and the columns are the samples.
+
+    Returns
+    -----------
+    pandas.DataFrame
+    """
+    expression_dataset = pd.read_table(path_to_file, header=0)
```

Comment:
Ah, so I had a problem here with `index_col=0` specifically on the TCGA dataset - I kept running out of memory when I specified the `index_col`. Tried looking into this more and couldn't get a satisfactory answer, but my laptop + a small EC2 instance can't handle whatever `pandas` is doing during that operation. :( It runs fine when I `set_index` separately, so even if it's not _great_ (in terms of style/verbosity) to do it this way, I'll probably leave it as is for ease of use